Tutorials/octave/octave_demo.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm.octave import *
+from qm.qua import *
+from configuration import *
+
+opx_ip = "xxx.xxx.xxx.xxx"
+opx_port = 80
+octave_ip = "xxx.xxx.xxx.xxx"
+octave_port = 80
+
+octave_config = QmOctaveConfig()
+octave_config.add_device_info("octave1", octave_ip, octave_port)
+
+octave_config.set_opx_octave_mapping([("con1", "octave1")])
+
+# If you want to use custom mapping you should use this function instead of the one above
+# octave_config.add_opx_octave_port_mapping(portmap)
+
+octave_config.set_calibration_db(os.getcwd())
+
+qmm = QuantumMachinesManager(host=opx_ip, port=opx_port, octave=octave_config)
+
+with program() as prog:
+    with infinite_loop_():
+        play("readout", "qe1")
+
+qm = qmm.open_qm(config)
+
+element = "q1"
+
+qm.octave.set_lo_frequency(element, lo_freq)
+qm.octave.set_rf_output_gain(element, -10)
+qm.octave.set_rf_output_mode(element, RFOutputMode.on)
+
+qm.octave.set_lo_source(element, OctaveLOSource.LO1)
+
+qm.octave.calibrate_element(element, [(lo_freq, if_freq)])
+qm = qmm.open_qm(config)  # Calibration closes the QM so another one should be opened again after calibration is done
```

Comment:
just realized that you have to open the qm twice.
very ugly :(

Reply:
you need to open one for each calibration you do.. it is very ugly and sadly there was no attention to it in recent months :(